packages/react-components/src/components/ChatMessage/MentionSuggestionRenderer.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+import { Mention } from '../MentionPopover';
+
+/**
+ * Provides the default implementation for rendering an Mention
+ * @param suggestion - The suggestion to render
+ *
+ * @beta
+ */
+export const defaultMentionSuggestionRenderer = (suggestion: Mention): JSX.Element => {
+  return (
+    <msft-mention id={suggestion.id} displayText={suggestion.displayText}>
+      {suggestion.displayText}
+    </msft-mention>
+  );
+};
+
+// Add the custom HTML tag to the JSX.IntrinsicElements interface
```

Comment:
Hmm, as a sdk, it looks not ideal to pollut global namespace environment, any alternative to avoid this?

Reply:
This one gives me 2 conflicts ideas in my mind:

1. If someone wants to use the custom html especially with typescript, then we will need this declaration to make it work, although it's not that standard
2. If someone doesn't need this one, they will see this 'msft-mention' when they try to write their own code in react (auto complete will show this one), after they import our package, it might be very confusing during development

Reply:
It's kinda like - 

1. if we don't pollute global namespace, then developers have to write their own .d.ts file for this custom tag
2. If we pollute global namespace, then it might be confusing for developers who don't know this feature

I am feeling react + typescript is not that ready for custom html tags

Reply:
Found a very neat way to do custom tag in react:
https://codepen.io/jiangnanhello/pen/vYVWVLv

Reply:
The global enables TS devs to use the new tag; we declare it here and is part of this feature and the SDK. As you pointed out, if we don't then any react user will need to create their own.
We've carefully chosen the name of the tag here to avoid clashing.

Reply:
Ok, I have a better solution. Will update....

Reply:
> Found a very neat way to do custom tag in react: https://codepen.io/jiangnanhello/pen/vYVWVLv

This doesn't compile in the codebase; it needs the extension to the JSX namespace still

Reply:
![image](https://user-images.githubusercontent.com/11863655/236284947-cf276b90-2212-4ce5-b770-3be50d8a239d.png)

This should be working well, validated locally, so we do all the type check for this small component props then wrap the custom tag as a react component (and always access it using that component)

The idea is modularized the custom tag, people should not see it unless they import the file